atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -243,7 +252,7 @@ public static TransactionManager createInMemory(Set<Schema> schemas) {
     }
 
     @JsonIgnore
-    @Value.Derived
+    @Value.Lazy
```

Comment:
read the method body! you can't make this change

Reply:
The method body is effectively 600-700 lines of wiring, what are you referring to exactly? It's also weird that just building a `TransactionManagers` starts kicking off everything which is what `@Value.Derived` does. Prior to the change you've suggested in this branch, it meant we couldn't complete it another way i.e. pass in the `Configurator` to the overloaded `serializable` method, because all the initialisation would have been kicked off by the time you've called `.build()`. Less of an issue with your suggested changes however.